src/Services/AppInstallActivationHandler.cs

```diff
@@ -51,63 +63,107 @@ protected override bool CanHandleInternal(ProtocolActivatedEventArgs args)
 
     protected async override Task HandleInternalAsync(ProtocolActivatedEventArgs args)
     {
-        await AppActivationFlowAsync(args.Uri.Query);
-    }
+        var uri = args.Uri;
+        var parameters = HttpUtility.ParseQueryString(uri.Query);
 
-    private async Task AppActivationFlowAsync(string query)
-    {
-        try
+        if (parameters != null)
         {
-            // Don't interrupt the user if the machine configuration is in progress
-            if (_setupFlowOrchestrator.IsMachineConfigurationInProgress)
+            var searchQuery = parameters.Get("search");
+            var wingetIdsQuery = parameters.Get("wingetIds");
+
+            if (!string.IsNullOrEmpty(searchQuery))
```

Comment:
we could optionally switch this from an if-else to support both `search` and `wingetIds` in the same URI.

Reply:
That would be a cool idea.  Is that operation valid?  Doing both in one request?

Reply:
I _think_ it would be valid but I'm realizing it would be more work than just this small change. We can pursue as an option down the road if there's a need for it, but I'm not going to include that as part of this PR.